output/mapping/eforms/mapping-tables/form-type.csv

```diff
@@ -1,7 +1,8 @@
 eForms code,eForms label,Release tag,Tender status
-planning,Procurement preparation and set-up.,'tender','planned'
+consultation,Consultation,'planning','planning'
+planning,Procurement preparation and set-up.,'tender;,'planning'
```

Comment:
I can't see this change discussed anywhere so I think it's just a typo?

```suggestion
planning,Procurement preparation and set-up.,'tender','planned'
```

Reply:
good spot, this was a typo